api/external/envoy/annotations/deprecation.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
```

Comment:
Moved this proto from Gloo into solo kit https://github.com/solo-io/gloo/pull/4777/files#r639104131